### ISSUE TITLE
Finish 2D Barcodes

### ISSUE THREAD

**Issue by samt** [samt/barcode#15]

**Comment by JanMikes** [comment]
Hey mate, im using your code and have to say it is pretty useful and working as intended.
Any chance to get this issue done?

**Comment by samt** [comment]
Yeah, it's been worked on. @Lexinek, just curious, are you more interested in QR or DataMatrix?

**Comment by JanMikes** [comment]
Definitely QR for me.

**Comment by samt** [comment]
Cool, I'm going to put this on my immediate roadmap. I've needed to finish this for some time now.

**Comment by ndugger** [comment]
👍Thumbs-up from me! Great work so far!

**Comment by tony1223** [comment]
i vote for this.
